packages/sommelier.rb

```diff
@@ -75,10 +75,9 @@ def self.build
           -Db_asneeded=false \
           -Db_lto=true \
           -Db_lto_mode=thin \
+          -Dwith_tests=false \
           -Dxwayland_path=#{CREW_PREFIX}/bin/Xwayland \
           -Dxwayland_gl_driver_path=/usr/#{ARCH_LIB}/dri -Ddefault_library=both \
```

Comment:
Use the `crew` version of `mesa`? (some ARM Chromebooks (like mine) do not come across with the `mesa` drivers preinstalled, all drivers are merged into the chrome binary instead)

Reply:
Ah yes, I suppose that makes sense!